apps/web-portal-client/lib/components/main-menu/MobileMenu.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import className from 'classnames'
+
+export function MobileMenu(props) {
+  const {items, menuStatus} = props
+
+  return (
+    <div
+      className={className({
+        'md:hidden': true,
+        block: menuStatus,
+        hidden: !menuStatus,
+      })}
+    >
+      <div className="px-2 pt-2 pb-3 space-y-1 sm:px-3">
+        {items.map((item) => (
+          <a
+            key={item.desc}
+            href={item.link}
+            className={className({
+              'block px-3 py-2 rounded-md text-base font-medium': true,
+              'text-white bg-gray-600': item.active,
+              'text-gray-300 hover:text-white hover:bg-gray-700': !item.active,
+            })}
+          >
+            {item.desc}
+          </a>
+        ))}
+      </div>
+    </div>
+  )
+}
+
+MobileMenu.propTypes = {
+  items: PropTypes.arrayOf(PropTypes.object),
+  menuStatus: PropTypes.bool,
```

Comment:
it seems like you don't need that prop because it's possible to handle on parent level like 
```
<div ...>
      {shouldShowForMobile && <MobileMenu items={...} />}
</div>

```

Reply:
Its not possible to handle showing menu on parents lavel because child contains classes that responsible for hiding / showing
```suggestion
   <div
      className={className({
        'md:hidden': true,
        block: menuStatus,
        hidden: !menuStatus,
      })}
    >
```
Actually we can left this Div element on the parent's level, but I think this logic - is natively logic of child

Reply:
Ok, then keep in component but give it more descriptive name, 'status' is too generic